tests/test_postprocess.py

```diff
@@ -55,6 +55,8 @@ def _make_jsonnet_filter(tmp_path, ns, enabled=None, create_namespace=False):
                     kind: "Namespace",
                     metadata: {
                         name: params.namespace,
+                        annotations: { },
```

Comment:
Why do you add the empty `annotations: {}` here?

Reply:
The test now uses the real `kube.libjsonnet` which adds the `annotations: { }`. Also reminded me to test if the shim is still needed and removed it: https://github.com/projectsyn/commodore/pull/694/commits/d7e038abe822f1ed3576902e8350dc6276fcc167

Reply:
It's done to be consistent between the built in helm_namespace and the custom jsonnet filter.

Reply:
Ah fair enough, might be worth it to add a comment explaining this here